docs/datamodel/understanding-datamodels.rst

```diff
@@ -67,8 +67,21 @@ Building a data model involves the following tasks:
 - Choose a persistent identifier scheme.
 - Configure the REST API and UI.
 
+Internal vs External representation:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The above mentioned tasks can be divided into two: Internal and external representation. 
+
+The two first ones, JSONSchemas and Elasticsearch mappings, correspond to the internal representation of the data. JSONSchemas are used to validate the data, verifying that it matches the Elasticsearch mapping. These last ones are used to specify the format in which the data is stored in Elasticsearch.
```

Comment:
> The two first ones, JSONSchemas and Elasticsearch mappings, correspond to the internal representation of the data.

The first two, where the JSONSchema and Elasticsearch mappings are defined, correspond to the internal representation of the data/records.